tasks/sign-base64-blob/tests/mocks.sh

```diff
@@ -23,3 +23,13 @@ function kubectl() {
 
   echo -n "dummy-payload" | base64
 }
+
+function gpg() {
+  if [ $* != "get internalrequest internal-request -o=jsonpath='{.status.results.signed_payload}'" ]
```

Comment:
I think this is a copy paste error as gpg is not called with these arguments in the task

Reply:
/hold You are right.

Reply:
Sure, setting up GPG keys would be the ideal test and I expect to do it in a future, but it's not something I can focus on right now.